sections/03_categorize.md

```diff
@@ -196,55 +196,73 @@ this field.
+Ratner, Bach, and Ré described training data as "The _New_ New Oil" [@url:http://hazyresearch.github.io/snorkel/blog/weak_supervision.html].
+Based on our review of research on deep learning methods to categorize disease,
+this rings true. We expect each of these approaches to play an important role
+if deep learning is going to transform how we analyze data to categorize states
+of human health. Finally, we don't expect that these methods will replace expert
+review. We expect them to complement expert review by allowing more efficient
+use of the costly practice of manual annotation.
 
 ###### "Right to explanation" laws and potential for discrimination
 
 In April 2016, the European Union adopted new rules regarding the use of
 personal information, the General Data Protection Regulation (GDPR)
-[@arxiv:1606.08813v3].  Among the new rules, it is perhaps best summed up in the
-phrase "right to explanation".   I.e., a clinician must be able to explain to a
-patient how a decision was reached by any machine learning algorithm that uses
-the patient's data.  The new rules were passed with the understanding that
-categorization or recommendation systems inherently profiles individuals, but
-may do so in a way that is unlawfully discriminatory.  
-
-On the other hand, as datasets become larger and more complex, medically
-important data relationships and correlations can become not only difficult to
-detect, but difficult for human understanding.  Machine learning and deep
-learning algorithms, which fixate solely on predictive accuracy, may be regarded
-as "black boxes" and are generally not tractable to human understanding.  
-
-Yet the power of machine learning is not disputed.  In the context of medical
-applications, machine learning holds the promise to save more lives and provide
-better quality of life for those it does save.  There is certainly no clear cut
-answer to balance patient rights and utilitarian gain, and these issues will
-only become more prominent as technology advances.  
+[@arxiv:1606.08813v3]. The new rules are perhaps best summed up by the phrase
+"right to explanation". Those who use machine learning algorithms must be able
+to explain how a decision was reached. For example, a clinician treating a
+patient who is aided by a machine learning algorithm may be expected to explain
+decisions that use the patient's data.  The new rules were designed to target
+categorization or recommendation systems, which inherently profile individuals,
+but can do so in ways that are discriminatory and unlawful `TODO: @traversc citation needed`.
+
+As datasets become larger and more complex, we may begin to identify
+relationships in data that are important for human health but difficult to
+understand. The algorithms described in this review and others like them may
+become highly accurate and useful for various purposes, including within medical
+practice. However, to discover and avoid discriminatory applications it will be
```

Comment:
There is a lot of work and discussion, not necessarily in the medical domain, on discrimination in machine learning.  Do we want to bring in external references?